uber/site_sections/summary.py

```diff
@@ -690,3 +691,39 @@ def checkins_by_hour(self, out, session):
     def all_attendees(self):
         raise HTTPRedirect('../export/valid_attendees')
     all_attendees.restricted = [c.ACCOUNTS and c.STATS and c.PEOPLE and c.MONEY]
+
+    def guidebook_exports(self, session, message=''):
+        return {
+            'message': message,
+            'tables': c.GUIDEBOOK_MODELS,
+        }
+
+    @csv_file
+    def export_guidebook(self, out, session, selected_model=''):
+        from uber.decorators import _set_response_filename
+
+        if 'Group' in selected_model:
```

Comment:
This is where all the ugly is contained. In an ideal world we wouldn't have all these special overrides, but in most cases, we want to add some type of custom filter to what we're exporting. That's all done here with as little code re-use as I could get away with.

There's a few possible improvements here -- I could make `has_been_accepted` a hybrid property and use a filter instead of list comprehension, for example, or improve the names of the exports (which are just things like `GuestGroup_bands_guidebook.csv`). I may do that before merging.